enterprise/api/v1/views.py

```diff
@@ -318,6 +318,7 @@ def enroll_learners_in_courses(self, request, pk):
                     catalog_api_user=request.user,
                     course_id=course_run,
                     users=pending_users | existing_users,
+                    bulk_enroll_mode=True,
```

Comment:
this allows us to use a different template without having to refactor most of the remaining code, in a backward compatible way